src/view/rooteditableelement.js

```diff
@@ -56,4 +55,16 @@ export default class RootEditableElement extends EditableElement {
 	set rootName( rootName ) {
 		this.setCustomProperty( rootNameSymbol, rootName );
 	}
+
+	/**
+	 * Overrides old element name and sets new one.
+	 * This is needed because name of view root has to be changed as the same as dom root name
+	 * when dom root is attached to view root.
```

Comment:
This is needed because view roots are created before they are attached to the DOM. The name of the root element is unknown at this stage. It has to be changed when the view root element is attached to the DOM element.